total_scattering/reduction/total_scattering_reduction.py

```diff
@@ -1097,8 +1135,11 @@ def TotalScatteringReduction(config=None):
         OutputWorkspace=van_corrected,
         SetError='zero')
 
-    # STEP 2.1: Normalize by Vanadium
-
+    #################################################################
+    # STEP 2.1: Normalize by Vanadium and
+    #           convert to Q (momentum transfer)
+    #           For sample, container, sample raw, vanadium background
+    #################################################################
```

Comment:
We should fix the following line,

[https://github.com/neutrons/mantid_total_scattering/blob/370564bd97bcc94239a69c8b0065673164954b51/total_scattering/reduction/total_scattering_reduction.py#L1158](https://github.com/neutrons/mantid_total_scattering/blob/370564bd97bcc94239a69c8b0065673164954b51/total_scattering/reduction/total_scattering_reduction.py#L1158)

to become like this,

```
# Save (sample - back) /  van_corrected
```